pymobiledevice3/remote/tunnel_service.py

```diff
@@ -515,15 +516,19 @@ async def _request_pair_consent(self) -> PairConsentResult:
         else:
             # On tvOS no consent is needed and pairing data is returned immediately.
             pairing_data = self._decode_bytes_if_needed(response['pairingData']['_0']['data'])
+            # On tvOS we need pin to setup pairing.
+            pin = input('Enter PIN: ')
```

Comment:
- this will request a pincode on all platforms (including ios)
- make so it will request a pincode ONLY for tvos

Reply:
The new commit has resolved it.